arcade/hitbox/base.py

```diff
@@ -8,6 +8,8 @@
 from arcade.types import Point, PointList
 
 
+__all__ = ["HitBoxAlgorithm", "HitBox", "RotatableHitBox"]
+
 class HitBoxAlgorithm:
```

Comment:
There should be an extra newline between these since they're top-level module members.
```suggestion
__all__ = ["HitBoxAlgorithm", "HitBox", "RotatableHitBox"]


class HitBoxAlgorithm:
```
The exceptions to this rule tend to be breaks between groups of imports or groups of top-level constants.